### PR TITLE
fix Python3 native on linux-wayland

### PR DESCRIPTION
**Pull request by wsnipex**
see commits

### CODE REVIEW

File: tools/depends/native/python36-native/Makefile [hunk 1]
```diff
@@ -22,7 +26,7 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
-	cd $(PLATFORM); $(CONFIGURE)
+	cd $(PLATFORM); $(EXPAT_FLAGS) $(CONFIGURE)
```
Comment (Rechi) [inline tools/depends/native/python36-native/Makefile]: @wsnipex leftover from testing?

File: tools/depends/native/python36-native/Makefile [hunk 2]
```diff
@@ -11,6 +11,10 @@ CWD=$(shell pwd)
 HOSTPYTHONDIR=$(CWD)/$(PLATFORM)/hostpython
 
 CONFIGURE=./configure --prefix=$(NATIVEPREFIX) --disable-shared --disable-framework --without-pymalloc
+ifeq ($(ENABLE_WAYLAND),yes)
+  CONFIGURE += --with-system-expat
```
Comment (Rechi) [inline tools/depends/native/python36-native/Makefile:15]: does passing this argument unconditionally causes an error?
If you want it explicit just ignore my comment.

File: tools/depends/native/python36-native/Makefile (hunk 1 above)
Comment (wsnipex) [reply tools/depends/native/python36-native/Makefile]: yep, good catch

File: tools/depends/native/python36-native/Makefile (hunk 2 above)
Comment (wsnipex) [reply tools/depends/native/python36-native/Makefile:15]: I prefer to make stuff explicit

**Review by Rechi — APPROVED** [review]

**Comment by arpitn30** [comment]
@wsnipex Can you please rebase and put the pull request again?

**Comment by wsnipex** [comment]
???

**Comment by arpitn30** [comment]
I accidentally pushed your changes away

**Comment by arpitn30** [comment]
Or just wait 5 min, I'll reset and put a pr from your branch myself

**Comment by Rechi** [comment]
@wsnipex shouldn't python27-native and python36-native depend on expat-native

**Comment by wsnipex** [comment]
oh yes.., only for wayland, forgot that.

**Pull request merged**